src/lib/components/molecules/canvas-map/lib/styles/Style.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * TODO: is this meant to be a subset of the style attr?
```

Comment:
@njdehoog am I right in thinking this class is meant to be a small subset of React's `CSSProperties` class? can we treat it as such? I gathered as much from how we get and set these styles in the map + layer code?

Reply:
Got it, thanks!